### PR TITLE
Constructors, Destructors and Assignment operators

### PR DESCRIPTION
**Pull request by elixir14082002**
<hr>

## Added documentation and video 📜

I have added documentation and video on Constructors, Destructors, and assignment operators.
#3035

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by elixir14082002** [comment]
@Suvraneel I have resolved the assigned issue. Please Verify!

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add the topic of multiple constructors. 
Add some more content to Assignment operators.

File: C_CPP/C++ 14/Constructor_Destructor.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+a = 10;
+a += 10; // This means a= a + 10;
+```
+
+There are several operators like:
+`=, +=, -=, *=, /=, %=, >>=, <<=, |=, &=, ^=`
+
+Let us understand the concepts with more clarity with the help of a video,
+
+[Video on Constructor and Destructor by elixir14082002](https://drive.google.com/file/d/15bq7ltmc7QIpRmBUtRmRSo15lXi7e2VX/view?usp=sharing)
```
Comment (kashishahuja2002) [inline C_CPP/C++ 14/Constructor_Destructor.md:216]: Change this line to "Video on Constructor and Destructor".

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

**Comment by elixir14082002** [comment]
@kashishahuja2002 I have made the changes. Kindly review!
@Suvraneel

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
The issue also has a topic of assignment operator. 
Cover that topic also in detail. Don't just add it for sake.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
The topic of multiple constructors should be before destructor.

**Pull request closed without merging**